frontend/src/scenes/billing/v2/test/Billing.tsx

```diff
@@ -75,161 +86,128 @@ export function BillingV2({ redirectPath = '' }: BillingV2Props): JSX.Element {
+                            disableClientSideRouting
+                            fullWidth
+                            center
+                        >
+                            Manage subscription
+                        </LemonButton>
+                    ) : showLicenseDirectInput ? (
+                        <>
+                            <Form
+                                logic={billingTestLogic}
+                                formKey="activateLicense"
+                                enableFormOnSubmit
+                                className="space-y-4"
                             >
-                                Manage subscription
-                            </LemonButton>
-                        ) : showLicenseDirectInput ? (
-                            <>
-                                <Form
-                                    logic={billingTestLogic}
-                                    formKey="activateLicense"
-                                    enableFormOnSubmit
-                                    className="space-y-4"
-                                >
-                                    <Field name="license" label={'Activate license key'}>
-                                        <LemonInput fullWidth autoFocus />
-                                    </Field>
-
-                                    <LemonButton
-                                        type="primary"
-                                        htmlType="submit"
-                                        loading={isActivateLicenseSubmitting}
-                                        fullWidth
-                                        center
-                                    >
-                                        Activate license key
-                                    </LemonButton>
-                                </Form>
-                            </>
-                        ) : (
-                            <div className="space-y-4">
+                                <Field name="license" label={'Activate license key'}>
+                                    <LemonInput fullWidth autoFocus />
+                                </Field>
+
                                 <LemonButton
-                                    to={`/api/billing-v2/activation?plan=${
-                                        enterprisePackage ? 'enterprise' : 'standard'
-                                    }&redirect_path=${redirectPath}`}
                                     type="primary"
-                                    size="large"
+                                    htmlType="submit"
+                                    loading={isActivateLicenseSubmitting}
                                     fullWidth
                                     center
-                                    disableClientSideRouting
                                 >
-                                    Setup payment Test
```

Comment:
This experiment is broken on self-hosted, as the button to upgrade has been removed here, but I don't think it's a big problem as long as we introduce a different upgrade table for self-hosted in a future iteration, as the experiment would never be visible to self-hosted so far.